indico/modules/events/registration/util.py

```diff
@@ -18,33 +18,33 @@
 
 from collections import OrderedDict
 
-from flask import current_app, session, json
+from flask import current_app, json, session
 from qrcode import QRCode, constants
-from sqlalchemy.orm import load_only, joinedload, undefer
+from sqlalchemy.orm import joinedload, load_only, undefer
 from werkzeug.urls import url_parse
 from wtforms import BooleanField, ValidationError
 
+from indico.core import signals
 from indico.core.config import Config
 from indico.core.db import db
 from indico.modules.events import EventLogKind, EventLogRealm
 from indico.modules.events.models.events import Event
 from indico.modules.events.registration import logger
-from indico.modules.events.registration.fields.choices import (ChoiceBaseField, AccommodationField,
+from indico.modules.events.registration.fields.choices import (AccommodationField, ChoiceBaseField,
                                                                get_field_merged_options)
-from indico.modules.events.registration.models.form_fields import (RegistrationFormPersonalDataField,
-                                                                   RegistrationFormFieldData)
+from indico.modules.events.registration.models.form_fields import (RegistrationFormFieldData,
```

Comment:
Isn't the `_` supposed to be sorted after the `s` character? (if so, swap line 35 and 37)